components/automate-ui/src/app/pages/client-runs/client-runs.component.html

```diff
@@ -4,7 +4,7 @@
 </div>
 
 <app-server-org-filter-sidebar>
-  <chef-sidebar-entry route="/client-runs" icon="storage">Node State</chef-sidebar-entry>
+  <!-- <chef-sidebar-entry route="/client-runs" icon="storage">Node State</chef-sidebar-entry> -->
```

Comment:
[nit] I'd rather just delete the line. git doesn't forget 😉

Reply:
i think you might be able to pull out the whole nav too, and not just that one entry.

Reply:
@srenatus I have removed the commented line. :)

Reply:
I'd merge, but I'm not sure I fully follow you there, @susanev: which _whole nav_ would you like to see pulled?

Reply:
yah, sorry i was not being clear. right now it renders a totally empty nav element when there are no items in the left-nav. i realize now, that workflow complicates things. the empty nav element can be annoying for people using screen readers. but its not important enough to fix in this pr i dont think. so ignore my comment, heh.